libs/hooks/README.md

```diff
@@ -17,68 +19,123 @@ Alternatively, you can also directly use the `RootComponentPropsProvider` requir
 :::
 
 **Example usage**
+
 ```tsx
 import { useRootComponentProps, RootComponentPropsProvider } from '@akashaorg/ui-core-hooks';
 
 const Component = () => {
-const {
-   baseRouteName,
-   uiEvents,
-   worldConfig,
-   getCorePlugins,
-   getTranslationPlugin,
-   navigateToModal,
-   // ...
-} = useRootComponentProps();
-
-// get i18n instance from the translation plugin
-const i18n = getTranslationPlugin().i18n
-
-// handle navigation using the routing from core plugins
-const handleNavigate = () => {
-  getCorePlugins().routing.navigateTo({
-    appName: 'applicaton namspace',
-    getNavigationUrl: () => {
-      // ...
-    }
-  })
-}
-}
+  const {
+    baseRouteName,
+    uiEvents,
+    worldConfig,
+    getCorePlugins,
+    getTranslationPlugin,
+    navigateToModal,
+    // ...
+  } = useRootComponentProps();
+
+  // get i18n instance from the translation plugin
+  const i18n = getTranslationPlugin().i18n;
+
+  // handle navigation using the routing from core plugins
+  const handleNavigate = () => {
+    getCorePlugins().routing.navigateTo({
+      appName: 'applicaton namspace',
+      getNavigationUrl: () => {
+        // ...
+      },
+    });
+  };
+};
 
-export default <RootComponentPropsProvider><Component/></RootComponentPropsProvider>
+export default (
+  <RootComponentPropsProvider>
+    <Component />
+  </RootComponentPropsProvider>
+);
 ```
-_________
+
+---
+
+### useAkashaStore
+
+Subscribe to changes in the application's authentication state, trigger login and logout actions, and access logged user's profile information
+
+> The useAkashaStore hook utilizes React's `useSyncExternalStore` hook to subscribe to a singleton instance of the `AuthenticationStore` class (under the hood), ensuring that the subscribing component automatically re-renders when the authentication state changes. The AuthenticationStore handles the logic for managing user authentication, by providing methods for logging in, logging out, restoring sessions, subscribing to state changes, and getting the current authentication state. Its static method `getInstance` ensures that only one instance of the AuthenticationStore exists (creates a new instance, if none exists).
```

Comment:
The description looks a bit redundant, can we refine it ? I tried to do it

The useAkashaStore hook provides access to the authentication state and authenticated user profile functionality. It subscribes to AuthenticationStore via React’s useSyncExternalStore, ensuring components update when authentication state changes. The AuthenticationStore handles authentication logic, including session management, state updates, and subscribes to authentication events from the global event channel.